src/main/java/pl/javagame/Player.java

```diff
@@ -0,0 +1,22 @@
+package pl.javagame;
+
+public class Player {
+
+    String name;
+
+    public Player(String name) {
+try {
+    this.name = name;
+    if (this.name == null || this.name=="")
+        throw new Exception ("Podaleś null lub pusty text");
+}
+        catch (Exception exception) {
```

Comment:
Tutaj troche mieszasz logikę "biznesową" z tą od "wyświetlania" danych. Tzn. pomyśl, że gdyby zamiast na terminal wyjątek musiałbym zostać wyświetlony 'w okienku', no to musielibyśmy tu dorzucić pokazywanie elementów graficznychm co nijak ma się do klasy Player.

Niech Player rzuci wyjątkiem, a najlepiej jakimś konrektnym - IllegalArgumentException w tym wypadku jest najlepszy, a niech zostanie obsłużony w głównej klasie aplikacji